tests/test_api_rooms.py

```diff
@@ -177,9 +177,36 @@ def describe_detail_next():
 
     def describe_POST():
 
-        def it_returns_200_and_updated_queue(client, populated_room):
-            status, content = load(client.post("/api/rooms/foobar/next"))
+        def it_returns_200_does_not_update_queue(client, populated_room):
```

Comment:
This is supposed to test the negative case which ignores the non-current speaker. I have moved it down and renamed the test.